src/api-reference/accounting-integration/v4.accountingintegration-vendor.markdown

```diff
@@ -13,6 +13,8 @@ This endpoint allows users to add, update, or delete ERP vendor data that is cac
 
 In addition, events will be published to notify API users of important updates. For more information, please see [Accounting Integration Events](/event-topics/event-subscription-topic-accountingintegration.html).
 
+Creating Vendors through this API will not automatically import those vendors into SAP Concur. Customers will need to add the accounting and employee vendors in SAP Concur via the provided UI screens.
```

Comment:
Hi @arjun-rathee-sap. Could you please move this to the Limitations section? Thank you.

Reply:
Done